packages/@react-spectrum/form/stories/Form.stories.tsx

```diff
@@ -150,6 +153,13 @@ function render(props: any = {}) {
         <Checkbox value="cats">Cats</Checkbox>
         <Checkbox value="dragons">Dragons</Checkbox>
       </CheckboxGroup>
+      <ColorField label="Primary Color" />
```

Comment:
May I refactor this to be one component of each type and in alphabetical order? Ditto the Provider update below.

Reply:
I mean, it's already mangling the chromatic story, so, i guess? Do be careful, form chromatic stories were already close to their maximum height
It falls into the same category as the other chromatic PR you have, we just need to do it in three steps very carefully
first run chromatic on main, make sure it's all green. then run it on this branch, make sure we're ok with everything. merge it, then run chromatic once more on main so we have the right baseline moving forward

Reply:
This sounds good. I was hoping that adding them at the end wouldn't cause too much chaos, but after reviewing, adding this many components makes the diff basically useless. :(

How do I tell what the max height is? Should the form stories be separated by categories like inputs and dropdowns?